config.yaml

```diff
@@ -10,13 +11,15 @@ permissions:
 commands:
   component:
     executable: /usr/bin/cog-command
+    description: Enumerate and view status of defined statuspage.io components
```

Comment:
I think we can leave the "statuspage.io" bit out of the command descriptions; I think it goes without saying 😄